developers/README.md

```diff
@@ -2,75 +2,65 @@
 
 ## Table of Contents
 
-* [Development Environment](#development-environment)
-  * [Virtual Machines](#virtual-machines)
 * [Contributing](#contributing)
-* [IRC](#irc)
+* [Discussion](#discussion)
+* [Local Development Environment](#local-development-environment)
+  * [Local Testing](#local-testing)
+* [Ansible Environments](#ansible-environmnets)
+  * [Production](#production)
+  * [Multinode Nodepool](#multinode-nodepool)
+  * [Docker](#docker)
+  * [Vagrant](#vagrant)
 
-## Development Environment
+## Contributing
 
-Before hacking on BonnyCI, we'll need to set up a development environment on our local machine.
+Please see the [contributing documentation](contributing) for more information.
 
-Supported development platforms:
+A brief note on repository naming...
+> The puns get made, or the plank gets walked.
+>
+> ~Jesse Keating
 
-* [Ubuntu](dev-environment/ubuntu.md)
-* [Apple's macOS](dev-environment/macOS.md)
+## Discussion
 
-### Virtual Machines
+For discussions about BonnyCI, join us on [freenode](https://freenode.net) #BonnyCi!
 
-You can test changes to BonnyCI locally by exercising some tools defined in [hoist](www.github.com/BonnyCI/hoist). Hoist is a set of ansible playbooks that automate the deployment of a BonnyCI environment. Be sure that you have completed the `Virtualization Tools` section of your OS's [Development Environment](#development-environment) page before proceeding.
+## Ansible Environments
 
-Clone hoist and navigate to its base directory:
+There are four different ansible environments that hoist supports:
 
-```shell
-$ git clone git@github.com:BonnyCI/hoist.git
-$ cd hoist
-```
+* [Production](#production)
+* [Multinode Nodepool](#multinode-nodepool)
+* [Docker](#docker)
+* [Vagrant](#vagrant)
 
-To perform a full deploy:
+### Production
 
-```shell
-$ vagrant up
-```
+This is the live system providing the active BonnyCI service. A cron job runs the hoist `install-ci.yml` playbook every 15 minutes.
```

Comment:
Since this documentation is intended for developers and not our admins, we should provide more information about how to *create* a production environment, not just document the basics of our existing one.